packages/gatsby-transformer-raml/src/utils/resource/examples-to-array.mjs

```diff
@@ -1,16 +1,7 @@
-import path  from "path";
-import fs  from "fs";
-
-
-export function resolveExampleFile(fileNodeDir, filePath) {
-  const exampleAbsolutePath = path.resolve(fileNodeDir, filePath);
-  return fs.readFileSync(exampleAbsolutePath, 'utf8');
-}
-
-export function examplesToArray(examples, fileNodeDir, resolveExampleFile) {
+export function examplesToArray(examples) {
   if (examples) {
     return Object.entries(examples).map(([name, value]) => {
-      const jsonString = resolveExampleFile(fileNodeDir, value.value);
+      const jsonString = JSON.stringify(value.value, null, 2);
```

Comment:
The current indentation of the json examples is not really ideal yet. I'm open to suggestions how we can improve that. Ideally it should not change the indentation. It should simply use the same indentation as in the generated raml file.

Reply:
@timonrey  the file based examples are formatted using prettier so you can use prettier to format the content if you want to ensure full consistency.     

I like moving to 2 space indentation to make it better readable. 

https://prettier.io/docs/en/api.html  and https://prettier.io/docs/en/options.html  -->  you can use prettier standards and only change the indentation depth if you want.